draft-ietf-quic-qpack.md

```diff
@@ -1011,7 +1016,7 @@ represented as an 8-bit prefix string literal.
 
 QPACK defines two settings which are included in the HTTP/3 SETTINGS frame.
 
-  SETTINGS_HEADER_TABLE_SIZE (0x1):
+  SETTINGS_MAXIMUM_DYNAMIC_TABLE_CAPACITY (0x1):
```

Comment:
I think we'd better have either "QPACK" or "HEADER" as part of the name, to avoid confusion. Other than that, the PR looks good to me.